bom/deployment/pom.xml

```diff
@@ -43,6 +43,15 @@
                 <scope>import</scope>
             </dependency>
 
+            <!-- Debezium -->
+            <dependency>
+                <groupId>io.debezium</groupId>
+                <artifactId>debezium-quarkus-outbox</artifactId>
```

Comment:
This has to be the deployment bom.

Reply:
@aloubyansky Just to be clear, setting this to point to https://github.com/Naros/debezium/blob/master/debezium-quarkus-outbox/deployment/pom.xml artifact work?

Reply:
Ah, hold on, that's not actually a BOM is it? It appears to be an actual extension artifact. So in that case you should remove `type` and `scope`. But still add `-deployment`, as it's a deployment BOM.
And do the same in the runtime bom.

Reply:
Alright, those are now covered in the latest pushes.